### DISCUSSION TITLE
What models are y'all using?

### DISCUSSION THREAD

**Discussion by acheong08** [acheong08/Diffusion-ColabUI#25] [Polls]
Just curious as to which are the more popular models

**Comment by wikibamxd** [comment]
i have a quick question, im new to all of SD and Novel AI can somoene tell me what running novel ai in stable diffusion web ui does different and also if in this colab there is the leaked version of novel ai, and i also dont know what VAE means.

**Reply by acheong08** [reply]
The different models creates different generations. NovelAI is for generating anime content. AnythingV3 is an improved version of NovelAI. The NovelAI here is the leaked NovelAI.

VAE improves the output by changing the weights. the 1.5 vae is good for general improvements while the Anime vae is for, well, anime.

**Reply by wikibamxd** [reply]
Is anything v3 better than the novel ai (leaked) here in the colab?

**Reply by acheong08** [reply]
AnythingV3's gens are generally better: Clearer lines, less deformed, better hands, etc. It's not as great for getting unique content but if that's what you're aiming for, WaifuDiffusion is probably better.
```
Customization < ---------------------------------------------- > Aestetics
WD1.3                               NovelAI                       AnythingV3 
```

**Reply by wikibamxd** [reply]
Ok ty , is the "wd1.3 " SD1.3?

**Reply by acheong08** [reply]
No. It's Waifu Diffusion 1.3

**Reply by acheong08** [reply]
Example from WD: 
![gen](https://user-images.githubusercontent.com/36258159/205278461-80f34799-acd1-401d-a294-2db76a09e895.png)

Example from AnythingV3:
![image](https://user-images.githubusercontent.com/36258159/205278516-0b820ce3-fee7-4147-81da-40276d17cea6.png)

**Reply by wikibamxd** [reply]
O damn ok ty
